### DISCUSSION TITLE
External Connection Attempt to my Wordpress container from 45.142.182.70

### DISCUSSION THREAD

**Discussion by stevenb123** [docker-library/wordpress#899] [General]
Hi,

I have a wordpress container running on a docker instance on my computer.  Today I had an external connection attempt from 45.142.182.70 that Norton created an alert for (and thankfully blocked).

Does anyone recognize this IP address?  Is it normal for a inbound connection on the wordrpess container from this IP address?  Any info would be appreciated.

Thanks in advance!

**Comment by tianon** [comment]
https://www.whois.com/whois/45.142.182.70 doesn't look familiar to me, but that doesn't mean much -- perhaps you've got a plugin that's doing some external access?

It's also worth considering whether your system might have been compromised and evaluating accordingly. :grimacing:
